src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const list = [];
+  let listItem = { ...state };
```

Comment:
What list and item is it? Variable name should describe it
[NAMING] - use proper names